src/keycloak/chart/templates/uds-package.yaml

```diff
@@ -73,6 +93,28 @@ spec:
         {{- end }}
       {{- end }}
 
+      - description: "Keycloak to Waypoint"
```

Comment:
Note to reviewers: These two seem to be the case for scraping metrics. Apparently Waypoint forwards the call to Keycloak.